lib/partition_setup.pm

```diff
@@ -326,9 +326,19 @@ sub addboot {
     }
 }
 
+sub skip_select_first_hard_disk {
+    return 1 if match_has_tag 'existing-partitions';
+    if (match_has_tag('select-hard-disks-one-selected')) {
+        send_key $cmd{next};
+        return 1;
+    }
+    return;
```

Comment:
Just to be consistent, let's return any value which denotes false here (optional).